src/util/index.ts

```diff
@@ -1,3 +1,9 @@
 export function classNames(...classes: any[]) {
   return classes.filter(Boolean).join(" ");
 }
+
+export function uint8ArrayToHexString(uint8Array: Uint8Array): string {
+  return Array.from(uint8Array)
+    .map((byte) => byte.toString(16).padStart(2, "0"))
+    .join("");
+}
```

Comment:
use the @stacks/common bytesToHex instead

Reply:
Updated to use @stacks/common